pages/Cart/components/Item/components/Coupon/components/Layout/components/CouponFreeShipping/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import I18n from '@shopgate/pwa-common/components/I18n';
+
+/**
+ * The Coupon Free Shipping component.
+ * @param {Object} props The component props.
+ * @returns {JSX}
+ */
+const CouponFreeShipping = ({ freeShipping }) => {
+  if (!freeShipping) {
+    return null;
+  }
+  return (
+    <span><I18n.Text string="cart.free_shipping" /></span>
```

Comment:
`I18n.Text` already wraps the text into <span> element. Maybe we could just return it?